frontend/src/app/core/global_search/input/global-search-input.component.html

```diff
@@ -1,14 +1,26 @@
-<div class="top-menu-search">
+<div class="top-menu-search" [attr.aria-label]="placeholder">
+
   <button
     *ngIf="expanded"
     (click)="toggleMobileSearch()"
     class="top-menu-search--back-button"
     href="#"
-    title="{{text.close_search}}"
+    [attr.title]="text.close_search"
+    aria-label="{{ text.close_search }}"
```

Comment:
Please use consistent syntax for binding attributes.

I wasn't sure if we really want to apply both `title` and `aria-label` to elements, but after doing a bit of reading I think it's ok *as long as those values are the same* and the button itself does not have any text. `title` provides a tooltip for mouse users and `aria-label` information for AT users.